### PR TITLE
Update INCORE docs according to changes for Interdependent recovery

### PR DESCRIPTION
**Pull request by ylyangtw**
Please refer to this pyincore PR https://github.com/IN-CORE/pyincore/pull/606 for changes.

### CODE REVIEW

File: manual_jb/content/analyses/housing_household_recovery.md [hunk 1]
```diff
@@ -32,11 +32,14 @@ key name | type | name | description
...
-`initial_stage_probability` <sup>*</sup> | [`incore:houseRecInitialStageFactors`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:houseRecInitialStageFactors) | Mass probability | Initial mass probability function for stage 0 of the Markov Chain.
+key name | type | name                        | description
+--- | --- |-----------------------------| ---
+`population_dislocation_block` <sup>*</sup> | [`incore:popDislocation`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:popDislocation) | Population dislocation      | Population dislocation results.
+`tpm` <sup>*</sup> | [`incore:houseRecTransitionProbMatrix`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:houseRecTransitionProbMatrix) | Probability matrix          | A transition probability matrix that specifies<br>the corresponding Markov chain per social vulnerability level.
+`initial_stage_probability` <sup>*</sup> | [`incore:houseRecInitialStageFactors`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:houseRecInitialStageFactors) | Mass probability            | Initial mass probability function for stage 0 of the Markov Chain.
```
Comment (navarroc) [inline manual_jb/content/analyses/housing_household_recovery.md:39]: This should be "initial_stage_probabilities" to match what's in the analysis spec

Comment (ylyangtw) [reply manual_jb/content/analyses/housing_household_recovery.md]: Thanks @navarroc I have addressed both

**Review by longshuicy — APPROVED** [review]

File: manual_jb/content/analyses/housing_household_recovery.md [hunk 2]
```diff
@@ -32,17 +32,20 @@ key name | type | name | description
 
 **Input datasets**
```
Comment (navarroc) [inline manual_jb/content/analyses/housing_household_recovery.md:33]: It looks like the parameter "num_cpu" is missing from the input parameters list for housing recovery sequential, can you add this?

Comment (ylyangtw) [reply manual_jb/content/analyses/housing_household_recovery.md:34]: Added num_cpu!

**Review by navarroc — APPROVED** [review]

**Pull request merged**